apps/financial-aid/web-veita/src/components/LogoMunicipality/index.tsx

```diff
@@ -37,7 +40,11 @@ const LogoMunicipality = ({ className }: LogoProps) => {
   })
 
   if (!data) {
-    return <LoadingDots />
+    return (
+      <Box>
+        <SkeletonLoader display="block" height={logoSize} width={logoSize} />
```

Comment:
Should this laod thought? Isn't this a constant once you are in?

Reply:
Yess once you are in but it takes few milliseconds to load at first and if loading dots are instead, it is very obvious 👀